core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -129,6 +129,11 @@ class KafkaApis(val requestChannel: RequestChannel,
       }
     )
 
+  // Not used. Just referencing singleton object(s) of breakdown metrics so the metrics are enforce created on start up.
+  val requestBreakdownMetrics: Map[ApiKeys, Object] = Map(
```

Comment:
Can we remove this block and still have the same functionality?

Reply:
Did an experiment and it works, so I remove it